content/src/roadmaps/raffle-bingo-steps/raffle-license-step-2.md

```diff
@@ -6,8 +6,8 @@ urlSlug: "raffle-license-step-2"
 name: "Apply for a Raffle or Bingo License in the NJ Municipality"
 webflowName: "Raffle or Bingo NJ Municipal License"
```

Comment:
[sand] There are a couple fields in here that are not being used and are not represented in the `config.yml` file. Let's remove these from here.

Reply:
Additionally, I'd opt to remove the `webflowId` from both of these. We're not going to be syncing these files to Webflow and this is just old data now.

Reply:
Both of these comments apply to both `raffle-license-step-1` and `raffle-license-step-2`